lib/plugins/inspec-sign/lib/inspec-sign/cli.rb

```diff
@@ -94,6 +94,8 @@ def generate_keys
         desc: "Path to profile directory"
       option :keyname, type: :string, required: true,
         desc: "Desriptive name of key"
+      option :profile_content_id, type: :string,
+        desc: "UUID of the profile. This will overwrite existing profile content id if already exist in the metadata file."
```

Comment:
"This will overwrite an existing profile content id if it already exists in the metadata file."